docs/version0.35.md

```diff
@@ -94,6 +95,12 @@ AIX OpenJ9 builds now require version 16.1.0.7 or later of the [IBM XL C++ Runti
 
 The `-XX:[+|-]JITServerLocalSyncCompiles` option is now enabled by default in most cases. For more information, see [`-XX:[+|-]JITServerLocalSyncCompiles`](xxjitserverlocalsynccompiles.md).
 
+### Unmarked phantom references are cleared immediately
```

Comment:
There is more that changed than just this topic. Rather than try to list them all, we could just refer to the maintenance spec for the list of changes.
i.e. The java.specification.maintenance.version system property, Reference.clone(), Reference.enqueue(), 
Runtime.runFinalizersOnExit() etc. are all relevant even though they didn't required changes elsewhere in the docs. Most of these required OpenJ9 changes to support them.